src/spec/verifiable-payid-protocol-NC.md

```diff
@@ -0,0 +1,379 @@
+      author:
+        ins: D. Fuelling
+        fullname: David Fuelling
+    PAYID-PROTOCOL:
+      title: "PayID Protocol"
+      author:
+         ins: A. Malhotra
+         fullname: Aanchal Malhotra
+         ins: D. Schwartz
+         fullname: David Schwartz
+
+informative:
+    RFC4732:
+
+--- note_Feedback
+
+  This specification is a draft proposal, and is part of the
+  [PayID Protocol](https://payid.org/) initiative. Feedback related to this
+  document should be sent in the form of a Github issue at:
+  https://github.com/payid-org/rfcs/issues.
+
+--- abstract
+
+This specification defines one of the extensions of the Basic PayID protocol [PAYID-PROTOCOL][] that aims to enable trust minimized PayID service with applications in non-custodial settings. More specifically, this extension of Basic PayID protocol eliminates the trust requirement between the PayID owner and their PayID service provider by allowing PayID server operators (such as wallets/exchanges) to send payment account(s) address information associated with a PayID [PAYID-URI][] that is digitally signed with the PayID private key of the PayID owner along with PayID owner's `identity` information and other meta-data needed to verify the signature.
+
+--- middle
+
+# Terminology
+   This protocol can be referred to as `Verifiable PayID Protocol for Non-Custodial Services`. It uses the following terminology.
+
+   * Endpoint: either the client or the server of a connection.
+   * Sender: individual or entity originating a transaction. 
+   * PayID client: the endpoint that initiates PayID protocol/sending side of the transaction.
+   * PayID server: the endpoint that returns payment account(s) address information in response to a PayID protocol request (non-custodial wallets, exchanges, etc).
+   * PayID owner: individual or entity receiving a transaction. 
+   * Digital Signture: As defined in [RFC4949][].
+
+   The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119][] and [RFC9174][].
+
+# Motivation
+
+   We anticipate that the most common use-case for retrieving the payment account(s) address information is to make transactions. Providers of such services can be categorized as custodial and non-custodial, each of which operate under different security models. Non-custodial wallets/exchanges do not store their customers’ on-ledger private keys on their servers. Instead, these customers hold their private keys and hence are in full control of their funds. As such, there is no trust requirement between non-custodial wallets/exchanges and their customers, and these services are not responsible for any for lost, compromised or stolen private keys of their customers. Likewise, customers of non-custodial wallets/exchanges do not need to worry if the servers of those wallets/exchanges go malicious or are compromised.
```

Comment:
Echoing this one-sentence change from the original PR -- I still think my suggestion is an improvement because us anticipating that "payments will be the most common use-case" doesn't really matter for this protocol/rfc. The efficacy of the protocol still stands if we were to assume some other common use-case, or none at all (also some minor grammatical edits in last few sentences).

```suggestion
   Providers of PayID-enabled payment services can be broadly categorized as custodial and non-custodial, each of which operate under different security models. Non-custodial wallets/exchanges do not store their customers’ on-ledger private keys on their servers. Instead, these customers hold their private keys and hence are in full control of their funds. As such, there is no trust requirement between non-custodial wallets/exchanges and their customers, and these services are not responsible for any for lost, compromised or stolen private keys of their customers. Likewise, customers of non-custodial wallets/exchanges do not need to worry if the servers of those wallets/exchanges are compromised.
```
```